cli/src/commands/extensions/mod.rs

```diff
@@ -311,3 +313,36 @@ pub fn installed_extensions() -> Result<Vec<Extension>> {
         })
         .collect::<Vec<_>>())
 }
+
+/// Get permissions based on CLI parameters.
+fn permissions_from_matches(matches: &ArgMatches) -> Permissions {
+    if matches.is_present("allow-all") {
+        return Permissions {
+            read: Permissions::new_read(&Some(Vec::new()), true),
+            write: Permissions::new_write(&Some(Vec::new()), true),
+            net: Permissions::new_net(&Some(Vec::new()), true),
+            env: Permissions::new_env(&Some(Vec::new()), true),
+            run: Permissions::new_run(&Some(Vec::new()), true),
+            ffi: Permissions::new_ffi(&None, false),
+            hrtime: Permissions::new_hrtime(false),
+        };
+    }
```

Comment:
Two comments:

1. Can we just use [`Permissions::allow_all()`](https://docs.rs/deno_runtime/0.71.0/deno_runtime/permissions/struct.Permissions.html#method.allow_all) here?
2. If they user sets `--allow-all`, we can imply they don't want to be prompted. So we should probably set `prompt` to `false`. (In theory this shouldn't matter... I just don't want to hang a script if something goes wrong)

Reply:
I'd say no to 1. because we want to explicitly deny the users from doing FFI. Dealing with that would be very tricky and the Deno team also warns the users about it in their [docs](https://deno.land/manual/getting_started/permissions):

> --allow-ffi Allow loading of dynamic libraries. Be aware that dynamic libraries are not run in a sandbox and therefore do not have the same security restrictions as the Deno process. Therefore, use with caution. Please note that --allow-ffi is an unstable feature.